Integrations/src/test/java/io/deephaven/integrations/learn/InputTest.java

```diff
@@ -2,28 +2,106 @@
 
 import io.deephaven.db.v2.InMemoryTable;
 import io.deephaven.db.v2.sources.ColumnSource;
-import org.jpy.PyObject;
-
-import junit.framework.TestCase;
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.function.Function;
+
 public class InputTest {
 
     @Test
-    public void input_constructorTest() {
-         InMemoryTable table = new InMemoryTable(
-            new String[]{"Symbols", "GroupedInts"},
-            new Object[]{
-                    new String[]{"A", "A", "AAPL", "AAPL", "AAPL", "B", "B", "B", "B"},
-                    new int[]{1, 2, 1, 2, 3, 1, 2, 3, 4}
-            });
-
-         String[] colNames = { "Symbols" };
-
-         //PyObject pyFunc =
-        Input pyInput = Input.createJavaInput(colNames, args -> args);
-        Assert.assertArrayEquals(table.select(colNames).getColumnSources().toArray(ColumnSource.ZERO_LENGTH_COLUMN_SOURCE_ARRAY), pyInput.createColumnSource(table));
+    public void createInputTest1() {
```

Comment:
These tests suffer from a few problems.
1) Names are not useful in understanding what is being tested.
2) DRY! Tons of repeated code, such as the table setup.  That should be in a `setUp` method or a function.
3) Not being mindful of the scope of variables.  `table` is created at the top and is not used until the bottom.

All of these things together make the unit tests very hard to follow.  This will be a challenge for the next guy, when they need to figure out what broke in 3 years.  It is hard for me to figure out what is being tested right now.